photon-core/src/main/java/org/photonvision/vision/processes/VisionSourceManager.java

```diff
@@ -40,20 +38,17 @@
 import org.photonvision.vision.camera.CameraQuirk;
 import org.photonvision.vision.camera.CameraType;
 import org.photonvision.vision.camera.LibcameraGpuSource;
-import org.photonvision.vision.camera.TestSource;
 import org.photonvision.vision.camera.USBCameras.USBCameraSource;
 
 public class VisionSourceManager {
     private static final Logger logger = new Logger(VisionSourceManager.class, LogGroup.Camera);
-    private static final List<String> deviceBlacklist = List.of("bcm2835-isp");
-
-    final List<CameraInfo> knownCameras = new CopyOnWriteArrayList<>();
 
-    final List<CameraConfiguration> unmatchedLoadedConfigs = new CopyOnWriteArrayList<>();
-    private boolean hasWarned;
-    private boolean hasWarnedNoCameras = false;
+    private static final List<String> deviceBlacklist = List.of("bcm2835-isp");
     private String ignoredCamerasRegex = "";
 
+    private final AtomicBoolean configsLoaded = new AtomicBoolean(false);
```

Comment:
What happens on first boot if no cameras are plugged in? Should we special case always matching the cameras that are connected at first boot, or the first camera ever plugged in?

Reply:
It should just load no configs right? So the user has to select the cameras.

Reply:
That sounds reasonable to me. or a modal, or big flashing light telling people to take action. Just want to make "why can't i see any cameras???!!! they're plugged in????" something that's intuitive for users to fix.